src/main/java/net/dv8tion/jda/core/entities/Invite.java

```diff
@@ -102,6 +102,17 @@
     String getCode();
 
     /**
+     * The invite URL for this invite in the format of:
+     * {@code "discord.gg/" + getCode()}
+     *
+     * @return Invite URL for this Invite
+     */
+    default String getURL()
+    {
+        return "discord.gg/" + getCode();
```

Comment:
I suppose that makes sense, was thinking for generally posting in channels but you're right.